common/app/renderers/DotcomRenderingService.scala

```diff
@@ -32,7 +35,7 @@ class DotcomRenderingService extends GuLogging with ResultWithPreconnectPreload
     resetTimeout = Configuration.rendering.timeout * 4,
   )
 
-  private[this] def get(
```

Comment:
Rename to `post` as is actually performing a `POST` request, not a `GET`, so previous name is misleading and could confuse.